<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="HW1121" name="com.mobile.automation.scripts.QantasDemo_ExtentReports" tests="1" failures="1" timestamp="02 Jan 2018 14:48:36 GMT" time="61.021" errors="0">
  <testcase name="Qantas_Demo" time="61.021" classname="com.mobile.automation.scripts.QantasDemo_ExtentReports">
    <failure type="java.lang.AssertionError" message="Test Case Failed because - Failed to Fill flight details">
      <![CDATA[java.lang.AssertionError: Test Case Failed because - Failed to Fill flight details
	at org.testng.Assert.fail(Assert.java:93)
	at com.mobile.automation.scripts.QantasDemo_ExtentReports.TestCaseStatus(QantasDemo_ExtentReports.java:32)
	at com.mobile.automation.scripts.QantasDemo_ExtentReports.Qantas_Demo(QantasDemo_ExtentReports.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:459)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- Qantas_Demo -->
</testsuite> <!-- com.mobile.automation.scripts.QantasDemo_ExtentReports -->
